doc/engsys_resource_management.md

```diff
@@ -0,0 +1,107 @@
+## Resource Management Guidelines
+
+This document contains guidelines for creating and managing Azure resources in the Azure SDK Engineering System
+subscriptions. EngSys has automation that will delete resources based on the criteria outlined below. This is to
+improve our overall security and keep us in compliance with company policies.
+
+  * [Managing Dev Resources in the Playground Subscription](#managing-dev-resources-in-the-playground-subscription)
+     * [Resource Groups](#resource-groups)
+     * [Role Assignments](#role-assignments)
+  * [Managing CI resources in the Test Subscription](#managing-ci-resources-in-the-test-subscription)
+     * [Resource Groups](#resource-groups-1)
+     * [Role Assignments](#role-assignments-1)
+
+## Managing Dev Resources in the Playground Subscription
+
+This section applies to resource groups located in the `Azure SDK Developer Playground` subscription.
+
+Currently, the automation will inspect resource groups and role assignments only.
+
+### Resource Groups
+
+The automation will delete all resource groups in the playground subscription DAILY, unless they meet at least one of the following criteria:
+
+- The resource group starts with a valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`, followed by an
+  optional hyphen and extra characters.
+    - Valid group name examples: `myalias`, `myalias-test-foobar` for `myalias@microsoft.com`
+- The resource group contains a tag with the name `Owners` where the value is a csv formatted string that contains at
+  least one valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`. This convention should only be used
+  when it is not possible to name the resource group with an alias (for example, inner groups auto-created by a resource
+  provider like AKS).
+    - Valid owner tag examples: `Owners: myalias`, `Owners: myalias,anotheralias,lastalias`
+- The resource group contains a tag with the name `DeleteAfter` and an [ISO8601 formatted date value](https://www.iso.org/iso-8601-date-and-time-format.html)
+  (see below for examples), where the date is greater than the current date and not greater than 7 days in the future.
+    - If the `DeleteAfter` value is in the past, or greater than 7 days in the future, it will be deleted.
+    - If you have a resource group for which you would like to extend the lifetime, update the `DeleteAfter` tag to a
+      future date to renew the lease.
+    - Resource groups which do not contain a `DeleteAfter` tag will have one added for a 24 hour duration
```

Comment:
I guess the opposite question is also interesting. If the resource has an owner and a DeleteAfter tag will we still honor the DeleteAfter tag?

Reply:
I should clarify here to say "Resource groups which do not contain a `DeleteAfter` or match at least one of the remaining naming/tagging criteria will have one added..."

Reply:
If it has an owner AND a tag, yes I think we should delete that. I can also clarify that, though I worry we now need a "cliffs notes" and a "full spec" because this is getting rather complicated to parse for someone who just wants to create a quick static resource. Maybe we just need a one sentence topline summary that says "if you want it to live forever, do X, if you need it for a few days, do Y."

Reply:
Yes I agree once we settle on the details we can add a summary based on the scenarios.